### PR TITLE
feat: add php support for models generation

### LINKED ISSUE #572: Add PHP model generation support
#### Reason/Context
Now that Modelina supports PHP model generation, we need to add support for it in the CLI.

To do this:
1. Add the language (`php`) to the list of languages: https://github.com/asyncapi/cli/blob/c2b5cb38ce9fb544139a6fab2da6a5639d018e29/src/commands/generate/models.ts#L9
1. Add a new switch entry as we have for JavaScript: https://github.com/asyncapi/cli/blob/c2b5cb38ce9fb544139a6fab2da6a5639d018e29/src/commands/generate/models.ts#L209
1. Add a new test case for the new language: https://github.com/asyncapi/cli/blob/c2b5cb38ce9fb544139a6fab2da6a5639d018e29/test/commands/generate/models.test.ts#L73

### PR DESCRIPTION
**Pull request by Shurtu-gal**
**Description**

- add php model generation support to cli in accordance to modelina

**Related issue(s)**
Fixes #572

### CODE REVIEW

**Review by jonaslagoni — DISMISSED** [review]
Awesome 👍 

Need a code owner to take a look as well 🙂

File: src/commands/generate/models.ts [hunk 1]
```diff
@@ -91,7 +92,7 @@ export default class Models extends Command {
      * C# specific options
      */
     namespace: Flags.string({
-      description: 'C# specific, define the namespace to use for the generated models. This is required when language is `csharp`.',
+      description: 'C# and Php specific, define the namespace to use for the generated models. This is required when language is `csharp` or `php`.',
```
Comment (derberg) [inline src/commands/generate/models.ts:95]:
```suggestion
      description: 'C# and PHP specific, define the namespace to use for the generated models. This is required when language is `csharp` or `php`.',
```

**Review by derberg — COMMENTED** [review]
one tiny thing to fix

**Comment by Shurtu-gal** [comment]
> one tiny thing to fix

Done @derberg

**Comment by derberg** [comment]
sorry but as a result of merging https://github.com/asyncapi/cli/pull/575 you have conflicts that you need to solve

**Comment by Shurtu-gal** [comment]
> sorry but as a result of merging #575 you have conflicts that you need to solve

Done @derberg

**Comment by derberg** [comment]
please fix tests

**Comment by Shurtu-gal** [comment]
> please fix tests

Done @derberg

**Comment by jonaslagoni** [comment]
@Shurtu-gal do you mind updating the docs as well? 

https://github.com/asyncapi/cli/blob/master/docs/usage.md

Just add PHP and since C++ has been merged, you can add `cplusplus` as well 🙂

**Review by derberg — APPROVED** [review]
soon there will be no need for manual usage docs updates ❤️ 
https://github.com/asyncapi/cli/pull/556

**Comment by derberg** [comment]
@allcontributors please add @Shurtu-gal for code

**Pull request merged**